pkgs/top-level/aliases.nix

```diff
@@ -1523,12 +1523,6 @@ mapAliases ({
 
   # LLVM packages for (integration) testing that should not be used inside Nixpkgs:
   llvmPackages_latest = llvmPackages_18;
-  llvmPackages_git = recurseIntoAttrs (callPackage ../development/compilers/llvm/git {
-    inherit (stdenvAdapters) overrideCC;
-    buildLlvmTools = buildPackages.llvmPackages_git.tools;
-    targetLlvmLibraries = targetPackages.llvmPackages_git.libraries or llvmPackages_git.libraries;
-    targetLlvm = targetPackages.llvmPackages_git.llvm or llvmPackages_git.llvm;
-  });
```

Comment:
This was added to aliases to prevent packages in nixpkgs from using it. Dropping it from aliases would change that. Is it intended that llvmPackages_git be usable in nixpkgs going forward?

Reply:
I don't really care if LLVM git is or isn't in alias. I just moved it so all versions past 12 are using the commonified default.nix. Probably isn't a bad idea either way so Ofborg tests for the weekly git updates.

Reply:
This was in aliases.nix on purpose — if packages in Nixpkgs start relying on it, it blocks us from keeping it up to date, and the point of llvmPackages_git is to always be up to date.  Could you please move it back?

Reply:
(For the same reason, e.g. linuxPackages_latest is in aliases.nix on purpose, because when it wasn't it created tension between keeping it up to date and keeping dependent packages working.)

Reply:
> linuxPackages_latest is in aliases.nix on purpose

Where is `linuxPackages_latest` in `aliases.nix`? I don't see it in there.

Reply:
Moved back into aliases.nix in #331880

Reply:
> Where is `linuxPackages_latest` in `aliases.nix`? I don't see it in there.

Sorry, I was mistaken about that.  It's not in aliases.nix — it's been prevented from being used in Nixpkgs entirely through consistent review, and providing an alternative linuxHeaders package that doesn't promise to be "latest", even though in practice we update it as soon as we can.